src/compiler/reverse.jl

```diff
@@ -118,7 +118,10 @@ function instrument(ir::IR)
     if isexpr(ex, :foreigncall, :isdefined)
       continue
     elseif isexpr(ex, :enter, :leave)
-      error("try/catch is not supported.")
+      error("""try/catch is not supported.
+            Refer to the Zygote documentation for fixes.
+            https://fluxml.ai/Zygote.jl/dev/limitations.html#Try-catch-statements-1
```

Comment:
I think this link causes https://github.com/FluxML/Flux.jl/issues/2015 

The correct link seems to be https://fluxml.ai/Zygote.jl/dev/limitations/#Try-catch-statements-1 , but I wonder if we should avoid automatic `#heading` things in links, as they will break if anyone touches the heading text.